skills/lark-slides/SKILL.md

```
@@ -0,0 +1,334 @@
+|------|------|
+| 需要了解详细 XML 结构 | [xml-format-guide.md](references/xml-format-guide.md) |
+| 需要 CLI 调用示例 | [examples.md](references/examples.md) |
+| 需要参考真实 PPT 的 XML | [slides_demo.xml](references/slides_demo.xml) |
+| 需要用 table/chart 等复杂元素 | [slides_xml_schema_definition.xml](references/slides_xml_schema_definition.xml)（完整 Schema） |
+| 需要了解某个命令的详细参数 | 对应命令的 reference 文档（见下方参考文档章节） |
+
+## Workflow
+
+> **这是演示文稿，不是文档。** 每页 slide 是独立的视觉画面，信息密度要低，排版要留白。
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)